ip_entry.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import Column, Integer, VARCHAR, DateTime
```

Comment:
No need for `VARCHAR`. Use `String` instead.

Reply:
OK. Will change. Why? Isn't it more wasteful?